File: examples-sdk/typescript/037-basic-vilmodel-derive.ts
#!/usr/bin/env tsx
// 037-basic-vilmodel-derive — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 037-basic-vilmodel-derive.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("insurance-claim-processing", 8080);
const claims = new ServiceProcess("claims");
claims.endpoint("POST", "/claims/submit", "submit_claim");
claims.endpoint("GET", "/claims/sample", "sample_claim");
server.service(claims);
server.compile();

File: examples-sdk/python/037-basic-vilmodel-derive.py
#!/usr/bin/env python3
"""037-basic-vilmodel-derive — Python SDK equivalent
Compile: vil compile --from python --input 037-basic-vilmodel-derive.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("insurance-claim-processing", port=8080)
claims = server.service_process("claims")
claims.endpoint("POST", "/claims/submit", "submit_claim")
claims.endpoint("GET", "/claims/sample", "sample_claim")
server.compile()
